comp/core/status/statusimpl/status_api_endpoints.go

```diff
@@ -48,13 +44,42 @@ func (s *statusImplementation) getStatus(w http.ResponseWriter, r *http.Request,
 		buff, err = s.GetStatus(format, verbose)
 	}
 
+	if len(buff) != 0 {
+		// scrub status output
+		s := scrubber.DefaultScrubber
+		var e error
+		if format == "json" {
+			buff, e = s.ScrubJSON(buff)
+		} else {
+			buff, e = s.ScrubBytes(buff)
+		}
+		if e != nil {
+			buff = []byte("[REDACTED] - failure to clean the message")
+		}
+	}
+
 	if err != nil {
+		var errorMsg string
+		var scrubbedMsg []byte
+		var scrubOperationErr error
+
 		if format == "text" {
-			http.Error(w, s.log.Errorf("Error getting status. Error: %v.", err).Error(), http.StatusInternalServerError)
-			return
+			errorMsg = fmt.Sprintf("Error getting status. Error: %v.", err)
+			scrubbedMsg, scrubOperationErr = scrubber.DefaultScrubber.ScrubBytes([]byte(errorMsg))
+		} else {
+			errorMsg = fmt.Sprintf("Error getting status. Error: %v, Status: %v", err, buff)
+			body, _ := json.Marshal(map[string]string{"error": errorMsg})
+			scrubbedMsg, scrubOperationErr = scrubber.DefaultScrubber.ScrubJSON(body)
 		}
 
-		SetJSONError(w, s.log.Errorf("Error getting status. Error: %v, Status: %v", err, buff), http.StatusInternalServerError)
+		if scrubOperationErr != nil {
+			scrubbedMsg = []byte("[REDACTED] failed to clean error")
+		}
+
+		http.Error(w, s.log.Error(string(scrubbedMsg)).Error(), http.StatusInternalServerError)
```

Comment:
In the places were we create the error message `[REDACTED] ...` we are not taking into account the type of `format. If there is an error when returning JSON we should return a valid json object, something like: 

```json
{
   "error": "[REDACTED] - ...."
}
```

This applies as well on line error message when scrubbing the status output